easybuild/tools/github.py

```diff
@@ -395,6 +407,235 @@ def post_comment_in_issue(issue, txt, repo=GITHUB_EASYCONFIGS_REPO, github_user=
         raise EasyBuildError("Failed to create comment in PR %s#%d; status %s, data: %s", repo, issue, status, data)
 
 
+def _copy_easyconfigs_to_repo(paths, target_dir):
```

Comment:
it's an indication that this function is to be considered private, i.e. shouldn't be imported in other places... it basically provides the code common to both `--new-pr` and `--update-pr`, but is pretty useless in a different context